index.html

```diff
@@ -40,7 +40,7 @@ <h2 id="main-h2">Private Digital Currency</h2>
                     <div class="row middle-xs info-block-row private">
                         <div class="col-lg-6 col-md-6 col-sm-8 col-xs-12  why-text">
                             <h3>Monero is secure.</h3>
-                            <p>Monero can't be hacked to steal your funds, due to the power of distributed consensus, which you can read about here. This means that you are responsible for your money, and don't have to trust any entity to keep it safe for you.</p>
+                            <p>Monero cannot be hacked to steal your funds, due to its decentralized nature. Only you are responsible for securing your Monero.</p>
```

Comment:
I agree with the first change, but the second sentence doesn't need to change. That's just my opinion.